apps/send-func/openapi/aar-notification.yaml

```
@@ -0,0 +1,460 @@
+        taxId:
+          title: Codice Fiscale
+          description: Codice Fiscale
+          type: string
+          example: CGNNMO01T10A944Q
+    HttpStatusCode:
+      type: integer
+      format: int32
+      description: The HTTP status code generated by the origin server for this occurrence of the problem.
+      minimum: 100
+      maximum: 600
+      exclusiveMaximum: true
+      example: 200
+    ProblemJson:
+      type: object
+      properties:
+        type:
+          type: string
+          format: uri
+          description: |-
+            An absolute URI that identifies the problem type. When dereferenced,
+            it SHOULD provide human-readable documentation for the problem type
+            (e.g., using HTML).
+          default: about:blank
+          example: https://example.com/problem/constraint-violation
+        title:
+          type: string
+          description: |-
+            A short, summary of the problem type. Written in english and readable
+            for engineers (usually not suited for non technical stakeholders and
+            not localized); example: Service Unavailable
+        status:
+          $ref: "#/components/schemas/HttpStatusCode"
+        detail:
+          type: string
+          description: |-
+            A human readable explanation specific to this occurrence of the
+            problem.
+          example: There was an error processing the request
+        instance:
+          type: string
+          format: uri
+          description: |-
+            An absolute URI that identifies the specific occurrence of the problem.
+            It may or may not yield further information if dereferenced.
+        errors:
+          type: array
+          minItems: 1
+          items:
+            properties:
+              code:
+                description: Internal code of the error, in human-readable format
+                example: >-
+                  PN_PARAMETER_TOO_LONG | PN_PARAMETER_TOO_SHORT |
+                  PN_DUPLICATE_ENTRY | etc...
+                type: string
+              element:
+                description: Parameter or request body field name for validation error
+                example: body.order.item[2].quantity
+                type: string
+              detail:
+                description: >-
+                  A human readable explanation specific to this occurrence of
+                  the problem.
+                example: Parameter not valid
+                maxLength: 1024
+                type: string
+            required:
+              - code
+        traceId:
+          description: Internal support identifier associated to error
+          example: 123e4567-e89b-12d3-a456-426614174000
+          type: string
+      required:
+        - type
+        - status
+        - description
+        - errors
+  securitySchemes:
+    ApiKeyAuth:
+      type: apiKey
+      in: header
+      name: x-api-key
+security:
+  - ApiKeyAuth: []
```

```suggestion
  securitySchemes:
    Bearer:
      type: apiKey
      name: Authorization
      in: header
security:
  - Bearer: []
```
can we use Bearer auth instead?

---

1cd5dd90620a4c657453f568dd953bf07cb3cb95